refactoring/token-replace.py

```diff
@@ -0,0 +1,77 @@
+#!/bin/env python3
+
+usageHelp = r"""
+
+Replace a given token string with another string in a file (but only touch the
+file if there were changes).
+
+This will only match complete tokens that match the regex:
+
+  ([^A-Za-z0-9_])|^)<token-to-replace)[^A-Za-z0-9_]
+"""
+
+
+def getCmndLineOptions():
+  from argparse import ArgumentParser, RawDescriptionHelpFormatter
+
+  clp = ArgumentParser(description=usageHelp,
+    formatter_class=RawDescriptionHelpFormatter)
+
+  clp.add_argument(
+    "-t", dest="tokenToReplace", required=True,
+    help="Token to repalce" )
+
+  clp.add_argument(
+    "-r", dest="replacementString", required=True,
+    help="Replacement string" )
+
+  clp.add_argument(
+    "-f", dest="inputFile", required=True,
+    help="Input file (and also output if -o <file> not specified)" )
+
+  clp.add_argument(
+    "-o", dest="outputFile", default="",
+    help="Input file (and also output if -o <file> not specified)" )
+
+  options = clp.parse_args(sys.argv[1:])
+
+  if options.outputFile == "":
+    options.outputFile = options.inputFile
+
+  return options
+
+
+#
+#  Main()
+#
+
+if __name__ == '__main__':
+
+  import sys, re
+
+  inOptions = getCmndLineOptions()
+
+  beginLineTokenPattern = re.compile(
+    r'^' + inOptions.tokenToReplace + r'([^A-Za-z0-9_])' )
```

Comment:
Using `f''` syntax may make this more readable.

Reply:
Is it possible to mix `r'...'` and `f'...'` in the same sting?

Reply:
FYI: I did a direct push to 'master' for this fix in commit https://github.com/TriBITSPub/TriBITS/commit/28a5c830dd51c488ca7ee299ff5fa6239f814c39.

There is no automated testing for that code so there is no point in creating a PR and running all of those GHA builds.